WalletWasabi.Fluent/Helpers/CoordinatorConfigStringHelper.cs

```diff
@@ -0,0 +1,91 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using NBitcoin;
+using WalletWasabi.Bases;
+using WalletWasabi.Daemon;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+public static class CoordinatorConfigStringHelper
+{
+	// Format: ccs|<network>|<endpoint>|<coordinatorFee>|<absoluteMinInputCount>|<readMore>
+	private const int ExpectedParts = 6;
+	private const char Separator = '|';
+	private const string Prefix = "ccs";
+
+	public static async Task ProcessAsync(CoordinatorConfigString coordinatorConfigString)
+	{
+		// TODO: Verify with GetStatus?
+
+		// TODO: SANITY CHECKS !!!
+
+		PersistentConfig config = ConfigManagerNg.LoadFile<PersistentConfig>(Services.PersistentConfigFilePath);
+
+		if (coordinatorConfigString.Network == Network.Main)
+		{
+			config = config with { MainNetCoordinatorUri = coordinatorConfigString.Endpoint.ToString() };
+		}
+		else if (coordinatorConfigString.Network == Network.TestNet)
+		{
+			config = config with { TestNetCoordinatorUri = coordinatorConfigString.Endpoint.ToString() };
+		}
+		else if (coordinatorConfigString.Network == Network.RegTest)
+		{
+			config = config with { RegTestCoordinatorUri = coordinatorConfigString.Endpoint.ToString() };
+		}
+		else
+		{
+			Logger.LogWarning("Unknown network");
+			return;
+		}
+
+		config = config with { MaxCoordinationFeeRate = coordinatorConfigString.CoordinatorFee };
+
+		// TODO: Implement AbsoluteMinInputCount and ReadMore
+
+		ConfigManagerNg.ToFile(Services.PersistentConfigFilePath, config);
```

Comment:
The values should be updated in the `ApplicationSettings` class, otherwise the UI will not detect the changes.